doc/controller-card_poweroff.md

```diff
@@ -0,0 +1,132 @@
+
+
+## Proposed Rules
+
+Idea here is that the subject controller-card can be disabled using the OC-Path [/components/component/controller-card/config/power-admin-state](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-platform.html#components-component-controller-card-config-power-admin-state) by setting its value to POWER\_DISABLED. Following rules:
+
+
+
+1. Only a CONTROLLER_CARD in state redundant-roleSECONDARY will honor config/power-admin-state set toPOWER_DISABLED. If the controller-card's'redundant-role' is 'PRIMARY', and it's 'config/power-admin-state' is set to 'POWER_DISABLED', the NOS must allow the configuration. However, the change must take effect only on the next reboot. Following scenarios in context
+             
+`Scenario#1:`
+
+Let's say controller-card0 is PRIMARY and controller-card1 is SECONDARY and controller-card1 receives an operator driven change of config/power-admin-state = POWER_DISABLED, then controller-card0 will power-off controller-card1 immediately.  The leaf state/power-admin-state for controller-card1 must also be POWER_DISABLED. The 'state/last-poweroff-reason/trigger' should show as USER_INITIATED. The NOS may optionally update /state/last-poweroff-reason/details. state/last-poweroff-time should record the time when the card was powered-off. For example:
+
+```
+/components/component[name=controller0]/state/redundant-role, PRIMARY
+/components/component[name=controller0]/controller-card/config/power-admin-state, POWER_ENABLED
+/components/component[name=controller0]/controller-card/state/power-admin-state, POWER_ENABLED
+/components/component[name=controller1]/state/redundant-role, SECONDARY
+/components/component[name=controller1]/controller-card/config/power-admin-state, POWER_DISABLED
+/components/component[name=controller1]/controller-card/state/power-admin-state, POWER_DISABLED
+/components/component[name=controller1]/state/last-poweroff-reason/trigger, USER_INITIATED
+/components/component[name=controller1]/state/last-poweroff-reason/details, "User initiated Shutdown"
+/components/component[name=controller1]/state/last-poweroff-time, 1706590714681765937
+```
+
+`Scenario#2:`
+    
+If controller-card0 is PRIMARY and controller-card1 is SECONDARY and if controller-card0 is set to config/power-admin-state = POWER_DISABLED by an operator, then controller-card0 will stay powered-on until the next reboot. The leaf state/power-admin-state must show as POWER_ENABLED. The state/last-poweroff-reason/trigger should show as USER_INITIATED. Also the NOS can update the leaf /state/last-poweroff-reason/details. The leaf state/last-poweroff-time should record the time when the card powers-off post the next reboot. For example: 
```

Comment:
```suggestion
If controller-card0 is PRIMARY and controller-card1 is SECONDARY and if controller-card0 is set to config/power-admin-state = POWER_DISABLED by an operator, then controller-card0 will stay powered-on until the next reboot. `state/power-admin-state` must show as POWER_ENABLED. If a reboot of the PRIMARY CONTROLLER_CARD occurs,  `state/last-poweroff-time` should record the time when the card powers-off, `state/last-poweroff-reason/trigger` should show as USER_INITIATED and `/state/last-poweroff-reason/details` may be updated. For example: 
```

Reply:
Trigger and and details should only be updated after a power-off occurs (after a reboot in this scenario)